dialogTemplate/fr.json

```diff
@@ -0,0 +1,806 @@
+                                "Baisse la lumière dans le {séjour:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+				"S'il te plait, baisse la lumière à {25%:=>Percent}",
+                                "Baisse la lumière à {25%:=>Percent} s'il te plait",
+                                "Baisse la lumière à {25%:=>Percent}",
+                                "Baisse la lumière s'il te plait",
+				"S'il te plait, Baisse la lumière"
+				"Baisse la lumière"
+				"Baisse les lumières de {partout:=>Room} à {50 pourcents:=>Percent}",
+                                "Baisse les lumières {partout:=>Room} à {50 pourcents:=>Percent}",
+                                "Baisse les lumières de la {chambre:=>Room} à {50 pourcents:=>Percent}",
+                                "Baisse les lumières dans la {chambre:=>Room} à {50 pourcents:=>Percent}",
+                                "Baisse les lumières du {séjour:=>Room} à {50 pourcents:=>Percent}",
+                                "Baisse les lumières dans le {séjour:=>Room} à {50 pourcents:=>Percent}",
+                                "S'il te plait, Baisse les lumières de {partout:=>Room} à {50 pourcents:=>Percent}",
+                                "S'il te plait, Baisse les lumières {partout:=>Room} à {50 pourcents:=>Percent}",
+                                "S'il te plait, Baisse les lumières de la {chambre:=>Room} à {50 pourcents:=>Percent}",
+                                "S'il te plait, Baisse les lumières dans la {chambre:=>Room} à {50 pourcents:=>Percent}",
+                                "S'il te plait, Baisse les lumières du {séjour:=>Room} à {50 pourcents:=>Percent}",
+                                "S'il te plait, Baisse les lumières dans le {séjour:=>Room} à {50 pourcents:=>Percent}",
+				"Baisse les lumières de {partout:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+                                "Baisse les lumières {partout:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+                                "Baisse les lumières de la {chambre:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+                                "Baisse les lumières dans {chambre:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+                                "Baisse les lumières du {séjour:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+                                "Baisse les lumières dans le {séjour:=>Room} à {50 pourcents:=>Percent} s'il te plait",
+				"S'il te plait, baisse les lumières à {25%:=>Percent}",
+                                "Baisse les lumières à {25%:=>Percent} s'il te plait",
+                                "Baisse les lumières à {25%:=>Percent}",
+                                "Baisse les lumières s'il te plait",
+				"S'il te plait, Baisse les lumières"
+				"Baisse les lumières"
+                                "Diminue la lumière de {partout:=>Room} à {50 pourcents:=>Percent}",
```

Comment:
We use tabs, never spaces: https://docs.projectalice.io/contribute/code-guidelines.html